Sources/Types.swift

```diff
@@ -194,6 +194,13 @@ public enum FitType {
     case vertically
 }
 
+@objc public enum ViewFilter: Int {
+    /// No filter, use all views
+    case none
```

Comment:
After more thinking, I think it would be clearer if we rename `none` for `all`
```suggestion
    /// Consider all views
    case all
```